spaceship/lib/spaceship/connect_api/models/build_beta_detail.rb

```diff
@@ -56,6 +56,14 @@ def ready_for_internal_testing?
       def ready_for_beta_submission?
         return external_build_state == ExternalState::READY_FOR_BETA_SUBMISSION
       end
+
+      def waiting_for_beta_review?
+        return [ExternalState::WAITING_FOR_BETA_REVIEW, ExternalState::IN_BETA_TESTING].include?(external_build_state)
```

Comment:
JW, what's the difference between `BETA_APPROVED` and `IN_BETA_TESTING`? I only found this https://developer.apple.com/documentation/appstoreconnectapi/internalbetastate which doesn't really document the possible values 😅

Reply:
Maybe it should be `IN_BETA_REVIEW` and not `IN_BETA_TESTING`?

Reply:
Good point @mattinasi, since this function is `waiting_for_beta_review`, it makes sense for it to be `IN_BETA_REVIEW` instead 👍